src/collective/ifttt/utils.py

```diff
@@ -84,7 +87,8 @@ def configure_rule(self, data):
         self.rule_id = storage.values()[-1].id
 
         # traverse to configuration page of content rule
-        rule_url = '/Plone/' + self.rule_id
+        portal = api.portal.get().__name__
+        rule_url = (os.sep).join(('', portal, self.rule_id))
```

Comment:
`os.sep` is no no for web paths. Nowadays it is theoretical that someone runs Plone on Windows, but if one does, `os.sep` is `’\`.

Reply:
Web is standard, so it should be safe to do `'/'.join()`.

If you want to use a helper, you could use

```
from posixpath import join as urljoin
c = urljoin(a, b)
```
as in https://stackoverflow.com/a/15279799

It uses `/` as separator on every OS.